### PR TITLE
Frontend disable

### PR DESCRIPTION
**Pull request by MarinPostma**
documents https://github.com/meilisearch/MeiliSearch/pull/1097, according to https://github.com/meilisearch/specifications/blob/master/text/0001-frontend-disable-prod.md

### CODE REVIEW

**Comment by curquiza** [comment]
I think we should merge this branch into the `prepare-meilisearch-12-2020` branch instead of the `master` one => https://github.com/meilisearch/documentation/pull/602
Right?

**Comment by tpayet** [comment]
Hey @react-learner, could you confirm these changes are sufficient and that we are not missing any other parts of the documentation where we could document that? (spoiler alert: it should not 🤞)

File: guides/advanced_guides/web_interface.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Web interface
 
-After adding documents to your MeiliSearch, it is possible to try out the search engine with the integrated web interface.
+After adding documents to your MeiliSearch, it is possible to try out the search engine with the integrated web interface. This only works in a [development environment](/guides/advanced_guides/configuration.md#options).
```
Comment (react-learner) [inline guides/advanced_guides/web_interface.md:3]:
```suggestion
After adding documents to your MeiliSearch, it is possible to try out the search engine with the integrated web interface. This only works in a [development environment](/guides/advanced_guides/configuration.md#environment).
```

**Review by react-learner — CHANGES_REQUESTED** [review]
Looks good.

**Pull request merged**